src/Core/MySQL/tests/gtest_MySQLGtid.cpp

```diff
@@ -0,0 +1,40 @@
+#include <gtest/gtest.h>
+#include <Core/MySQL/MySQLGtid.h>
+
+using namespace DB;
+
+
+GTEST_TEST(GTIDSetsContains, Tests)
+{
+    GTIDSets gtid_set,
+             contained1, contained2, contained3, contained4, contained5,
+             not_contained1, not_contained2, not_contained3, not_contained4, not_contained5, not_contained6;
+
+    gtid_set.parse("2174B383-5441-11E8-B90A-C80AA9429562:1-3:11:47-49, 24DA167-0C0C-11E8-8442-00059A3C7B00:1-19:47-49:60");
```

Comment:
This unit test contains incorrect UUIDs.

Reply:
Should we try to find correct ones instead?

Reply:
I already changed it, see the master branch.